src/main.c

```diff
@@ -2437,6 +2442,12 @@ void handle_movement(double dt) {
                 dy = 8;
             }
         }
+        /**This condition is used to implement logic for instant high fly feature.*/
+        if (glfwGetKey(g->window, CRAFT_KEY_HIGH_FLY)) {
+            if (g->flying){
+                vy = CRAFT_KEY_HIGH_FLY_HEIGHT;
```

Comment:
The _craft key_ name does not sound right & is confusing for someone who is first looking at the variable name. A more understandable name would be "CRAFT_KEY_FLY_HIGH".

Reply:
change done.